apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -93,7 +119,11 @@ export default function UserThreadPanel({
   }, [isOpenPopup])
 
   useEffect(() => {
-    if (threads && threads.filter((t) => t.threadId === activeThread?.threadId).length) return
+    if (
+      threads &&
+      threads.filter(t => t.threadId === activeThread?.threadId).length
+    )
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)